app/queries/app/global.ts

```diff
@@ -21,20 +21,48 @@ export const getDeviceToken = async (appDatabase: Database): Promise<string> =>
     }
 };
 
-export const observeMultiServerTutorial = (appDatabase: Database) => {
-    return appDatabase.get<GlobalModel>(GLOBAL).query(Q.where('id', GLOBAL_IDENTIFIERS.MULTI_SERVER_TUTORIAL), Q.take(1)).observe().pipe(
+export const queryGlobalValue = (key: string) => {
+    const {database} = DatabaseManager.getAppDatabaseAndOperator();
+    return database.get<GlobalModel>(GLOBAL).query(Q.where('id', key), Q.take(1));
```

Comment:
Try/catch needed

Reply:
Added, but not so sure this is the best approach, since it adds some noise to the other helper functions. Also, not sure if we ever get to this point in the app if the app database is not present, but I guess is better be safe than sorry.